config/billing_rates.yml

```diff
@@ -0,0 +1,20 @@
+- id: 139d9a67-8182-8578-a303-235cabd5161c
```

Comment:
```
- { test1: 1, test2: 2 }
- { test1: 3, test2: 4 }
```
Online checkers say its valid YAML. Can you check with Ruby parser? (I'm at outside couldn't validate with my computer atm). Single line billing rates looks easy to maintain, compare etc.

Reply:
I tried at plane, it looks like working.